javascript/olm_pre_node.js

```
@@ -0,0 +1,28 @@
+var nodeCrypto = require("crypto");
+var get_random_values = function (buf) {
+    var bytes = nodeCrypto["randomBytes"](buf.length);
+    buf.set(bytes);
+};
+
+/* applications should define OLM_OPTIONS in the environment to override
+ * emscripten module settings
+ */
+if (typeof OLM_OPTIONS !== "undefined") {
+    for (var olm_option_key in OLM_OPTIONS) {
+        if (OLM_OPTIONS.hasOwnProperty(olm_option_key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)